LinuxFunctionsStacks.json

```diff
@@ -185,6 +185,40 @@
                 "isDeprecated": null
             }
         },
+        {
+            "id": null,
+            "name": "Custom",
+            "type": "Microsoft.Web/availableStacks?osTypeSelected=LinuxFunctions",
+            "properties": {
+                "name": "Custom",
+                "display": "Custom",
+                "dependency": null,
+                "majorVersions": [
+                    {
+                        "displayVersion": "custom",
+                        "runtimeVersion": null,
```

Comment:
Note: On Linux - linuxFxVersion should not be set

Reply:
Yeah this would require a change to the CLI. I believe we had decided a linux app would always have a linuxFxVersion, so the CLI assumes it will be there. Created an issue [here](https://github.com/Azure/Azure-Functions/issues/1673)

Reply:
@ammanifold  - please do review this PR as well and ACK for linuxFxVersion null